public/app/features/trails/ActionTabs/MetricOverviewScene.tsx

```diff
@@ -110,7 +112,7 @@ export class MetricOverviewScene extends SceneObjectBase<MetricOverviewSceneStat
             </Stack>
             <Stack direction="column" gap={0.5}>
               <Text weight={'medium'}>
-                <Trans i18nKey="trails.metric-overview.labels-label">Labels</Trans>
+                {useOtelExperience ? <Trans i18nKey="">Metric attributes</Trans> : <Trans i18nKey="">Labels</Trans>}
```

Comment:
I commented in the translation json file but want to point here too. I think the keys must be in snake-case